doc/onboarding.md

```diff
@@ -3,6 +3,12 @@
 This document is an outline of the things we tell new Collaborators at their
 onboarding session.
 
+## One week before the onboarding session
+
+* Ask the new Collaborator if they are using two-factor authentication on their GitHub account. If they are not, suggest that they enable it as their account will have elevated privileges in mamy of the Node.js repositories.
```

Comment:
Perhaps "strongly suggest". I'd even prefer "wait until they've enabled" but this is probably a discussion that lives someplace else.